Base/usr/share/man/man1/allocate.md

```diff
@@ -5,36 +5,40 @@ allocate - allocate memory
 ## Synopsis
 
 ```**sh
-$ allocate [number [unit (B/KiB/MiB)]]
+$ allocate [--unit B/KiB/MiB/GiB] [--sleep-time N] [number]
 ```
 
 ## Description
 
-`allocate` allocates a specific amount of virtual memory (specified in `number` and `unit`, by default 50 MiB), It also writes to each allocated page and then sleeps for 10 seconds. It is primarily used to test the kernel's memory management capabilities.
+`allocate` allocates a specific amount of virtual memory. If nothing is specified
+then it will allocate 100 MiB of memory.
+If `number` is specified without `unit`, it will default to `number` of bytes.
+It also writes to each allocated page and then sleeps for N seconds (by default 10).
+It is primarily used to test the kernel's memory management capabilities.
 
-## Arguments
+## Options
 
-* `number`: A number of `units` to allocate; the default is **50**
-* `unit`: Data size unit, can be `B` (bytes), `KiB` (kibibytes) or `MiB` (mebibytes); the default is **MiB**
+* `-u`, `--unit`: Storage Unit (Base 2 units - Bytes, KiB, MiB or GiB)
+* `-n`, `--sleep-time`: Number of seconds to sleep before freeing memory
 
 ## Examples
 
 ```sh
-$ allocate 100 MiB
-allocating memory (104857600 bytes)...
-done in 13ms
+$ allocate 500
```

Comment:
These examples don't work - you've set the default unit to `Bytes`, so this would allocate 500 bytes.